test/e2e/cross_os/test_job_submissions.py

```diff
@@ -1332,17 +1424,16 @@ def check_percentage(complete_percentage):
                         complete_percentage = session_action["progressPercent"]
                         return complete_percentage == 100
 
-            else:
-                return False
+            return False
 
         assert check_percentage(complete_percentage)
 
-        output_path: dict[str, list[str]] = wait_for_job_output(
+        output_path: Dict[str, List[str]] = wait_for_job_output(
```

Comment:
Why the change to Python 3.7 syntax for the typehints? The agent requires at least Python 3.9, so the previous typehints are just fine.

Reply:
Ah, this is just the syntax that I was used to, in the other code, so I wanted to be consistent. I could change it back if needed